test/org.eclipse.elk.graph.json.test/src/org/eclipse/elk/graph/json/test/EdgeCoordsTest.xtend

```diff
@@ -0,0 +1,175 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Kiel University and others.
+ * 
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package org.eclipse.elk.graph.json.test
+
+import com.google.gson.JsonParser
+import org.eclipse.elk.core.RecursiveGraphLayoutEngine
+import org.eclipse.elk.core.util.BasicProgressMonitor
+import org.eclipse.elk.core.util.Maybe
+import org.eclipse.elk.graph.json.ElkGraphJson
+import org.eclipse.elk.graph.json.JsonImporter
+import org.junit.Test
+
+import static org.junit.Assert.*
+
+/**
+ */
+class EdgeCoordsTest {
+    
```

Comment:
```suggestion
package org.eclipse.elk.graph.json.test

import com.google.gson.JsonParser
import org.eclipse.elk.alg.test.PlainJavaInitialization
import org.eclipse.elk.core.RecursiveGraphLayoutEngine
import org.eclipse.elk.core.util.BasicProgressMonitor
import org.eclipse.elk.core.util.Maybe
import org.eclipse.elk.graph.json.ElkGraphJson
import org.eclipse.elk.graph.json.JsonImporter
import org.junit.BeforeClass
import org.junit.Test

import static org.junit.Assert.*

/**
 */
class EdgeCoordsTest {
        
    @BeforeClass
    static def void init() {
        PlainJavaInitialization.initializePlainJavaLayout
    }
```

Reply:
So it is not a Java/Xtend issue. You could not directly import `LayeredMetaDataProvider` since the MANIFEST.MF in the test plugin does not contain a dependency to the layered plugin. The correct way is hence to use the initialization method from alg.test instead.